### ISSUE TITLE
[feature suggestion] existing winlib files are x86. if possible please use x64 version of it

### ISSUE THREAD

**Issue by eladkarako** [uup-dump/converter#27]
you're using that to compress the LZMS (MS LZMA),  
it's pretty much a 7zip client,  
using the x86 version, working memory is limited to top 2GB of working RAM,  
excessive paging to disk and back grinds the CPU is almost painful to watch..

if possible please update to use the x64 version.  
you can use their official one which does not include an embedded manifest and works kind of slow (even the x64 version),  

or take the amd64 build from here:  
https://github.com/eladkarako/winlib_patched_for_w10_w11/archive/refs/heads/master.zip

if you'll use mine you'll get extra speed because I've included an embedded manifest into the exe and dll that allows Windows to use segmented heap, so memory will be more efficiently allocated,  
and I also included a Windows 7,8,8.1,10,11 compatibility segment,  
so the exe and dll will not run through vista virtualization engine sandbox that all exe that does not have a compatibility section in its embedded manifest run through.  
meaning the exe can use the full potential of the OS, including more efficient memory allocation, large page (2MiB pages instead of 4KiB).

<hr/>

there is a x86 build as well,  
binary patched to be able to allocate 2GB of ram as well.  
you can use that for users of x86 Windows.

<hr/>

the better way is to have both exe and check `%PROCESSOR_ARCHITECTURE%` if it is `AMD64` to be able to run the x64 exe. this way you'll have compatibility for either type of users.

**Comment by abbodi1406** [comment]
`bin\bin64\wimlib-imagex` is x64 and used on Windows x64
`bin\wimlib-imagex` is used on Windows x86 or Arm64

not sure if your patched binary is safe to use on all Windows versions
maybe you could propose the changes to wimlib author

**Comment by eladkarako** [comment]
yes I've noticed the x64 bin,  
and the command launch the exe from the x64 bin as it should.

but for some reason the exe provided with the package was substantially different than the one provided in the official website.  
and it was able to allocated ~2GB of RAM and not more as you'll expect from the x64 binary (and was pretty much always on 50%) which hinted me it was probably a x86 build.  

I first replaced it locally with the original one,  
but then I've went one step further and added a proper manifest to allow it to opt-in to the new Windows 10 segmented heap and to be able to run outside of the vista virtualization engine that limits the performances of exe that do not have the compatibility segment in their embedded manifest.  

as a result,  
<img src="https://user-images.githubusercontent.com/415238/187301390-3016b4a0-c025-4cbc-a80e-5959cdec2ba9.png">  

I'm being able to allocate a lot of RAM,  
which helps with the compression,  
and reduce paging to disk and back.  

(I totally understand not to include a custom build into the converter though).
